exporter/datadogexporter/traces_exporter.go

```diff
@@ -226,7 +226,7 @@ func newTraceAgentConfig(ctx context.Context, params exporter.Settings, cfg *Con
 			return clientutil.NewHTTPClient(cfg.ClientConfig)
 		}
 	}
-	if v := cfg.Traces.flushInterval; v > 0 {
+	if v := cfg.Traces.FlushInterval; v > 0 {
```

Comment:
Is it fine to make this public? We were not exposing this to end-users before, we should error out if this is present in the `Unmarshal` function